builder/vsphere/iso/step_create.go

```diff
@@ -94,17 +94,18 @@ type StepCreateVM struct {
 func (s *StepCreateVM) Run(_ context.Context, state multistep.StateBag) multistep.StepAction {
 	ui := state.Get("ui").(packer.Ui)
 	d := state.Get("driver").(*driver.Driver)
+	vmPath := fmt.Sprintf("%s/%s", s.Location.Folder, s.Location.VMName)
```

Comment:
Same question here regarding the path separator. Since this is happening more than once, what do you think about adding a method on [LocationConfig](https://github.com/hashicorp/packer/blob/daffd9c3181d71253c27c7d59773d9e6ac17b206/builder/vsphere/common/config_location.go#L8) that generates the VMPath, as opposed to doing this in any of the steps that would require the full VM path?

Reply:
I think its not needed in anywhere else besides these two steps.

Is LocationConfig supposed to represent a Path in itself? if so, maybe it makes sense to put it there, but I'm not sure about its semantics, for example, a FullPath() method would return what? Maybe not only the folder+path, it maybe would also return full a vsphere inventor object path (what they call an ipath), and that might not work with FindVM. So I'm not sure what to name that method nor its semantics.

Reply:
> Is LocationConfig supposed to represent a Path in itself?

That's a good question and one that I don't know the answer to off the top of my head. I think holding on this change is a good move for now, and we can revisit in the future if there ever becomes a need.